scripts/dbSNP/format_dbSNP_GenVarSource_enum_schema.py

```diff
@@ -0,0 +1,69 @@
+'''
+Author:  Samantha Piekos
+Date:  09/30/20
+Name:  format_dbSNP_GevVarSource_enum_schema.py
+Description: Write all unique population sources recording genetic variant frequencies for a .vcf
+or .txt input file as enums of class GenVarSourceEnum to mcf output file.
+
+@file_input		input .vcf or .txt file of genetic variants
+@file_output	mcf output file recording all populations of genetic variant frequencies as enums of GenVarSourceEnum
+'''
+
+import sys
+
+def collect_freq_pop(freq_pops, set_freq_pop):
+	'''
+	For each genetic variant instance add all populations for its frequencies to a set.
+	@freq_pops		string of all populations recording frequencies for a genetic variant instance
+	@set_freq_pop	set of all populations recording genetic variant frequencies
+	@return			set of all populations recording genetic variant frequencies
+	'''
+	pops = freq_pops.split('|')
```

Comment:
What is the indent spacing in the script?  I think this repo uses 4 spaces as one indent.